src/components/InputImage/InputImage.tsx

```diff
@@ -0,0 +1,28 @@
+import { useState } from 'react';
+
+import scss from './InputImage.module.scss';
+
+function InputImage() {
+  const [imageSelected, setImageSelected] = useState(null);
+
+  function handleFileChange(event: any) {
+    const selectedImage = event.target.files[0];
+    setImageSelected(selectedImage);
+    // eslint desabilitado aqui para mostrar dados da imagem durante o desenvolvimento
+    // eslint-disable-next-line no-console
+    console.log('imagem', selectedImage);
+  }
+
+  return (
+    <div>
+      <div className={scss.button}>
+        <input type="file" onChange={handleFileChange} />{' '}
```

Comment:
Vale tirar esse espaço, geralmente o lint faz isso as vezes, ai pode tirar.



```suggestion
        <input type="file" onChange={handleFileChange} />
```